hack/update-ui.sh

```diff
@@ -44,11 +44,13 @@ shopt -s extglob
 
 rm -rf \
   !(web|Makefile*|scripts|VERSION) \
-  web/!(ui) web/ui/!(react-app|static) web/ui/static/* web/ui/react-app/.gitignore \
-  scripts/!(build_react_app.sh)
+  web/!(ui) \
+  web/ui/*.go \
+  web/ui/static/* web/ui/react-app/.gitignore \
+  scripts/!(compress_assets.sh)
 
 # Ensure empty target dir is committed.
 touch web/ui/static/.gitignore
 
-# Download modules according to lock file.
-cd web/ui/react-app && yarn install --frozen-lockfile
\ No newline at end of file
+# Install node_modules for vendoring.
+make ui-install
```

Comment:
Did you run `./hack/update-ui.sh v2.35.0` manually to bring in some of these changes?

Reply:
Yes